src/app/pages/PerpetualPage/components/RecentTradesTable/index.tsx

```diff
@@ -0,0 +1,52 @@
+import { AssetSymbolRenderer } from 'app/components/AssetSymbolRenderer';
+import { translations } from 'locales/i18n';
+import React from 'react';
+import { Trans } from 'react-i18next';
+import { PerpetualPair } from 'utils/models/perpetual-pair';
+import { usePerpetual_RecentTradesTable } from '../../hooks/usePerpetual_RecentTradesTable';
+import styles from './index.module.scss';
+import { RecentTradesTableRow } from './components/RecentTablesRow/index';
+
+type IRecentTradesTableProps = {
+  pair: PerpetualPair;
+};
+
+export const RecentTradesTable: React.FC<IRecentTradesTableProps> = ({
+  pair,
+}) => {
+  const data = usePerpetual_RecentTradesTable();
+
+  return (
+    <table className={styles['recent-trades-table']}>
```

Comment:
use camelCase
styles.recentTradesTable

Reply:
I will in this case, but don't we want to use BEM when we are writing styles?

Reply:
BEM is not needed with Module SCSS, since classes are hashed anyway.